code/modules/shuttle/emergency.dm

```diff
@@ -166,12 +166,36 @@
 		shuttle_master.emergencyLastCallLoc = null
 	emergency_shuttle_recalled.Announce("The emergency shuttle has been recalled.[shuttle_master.emergencyLastCallLoc ? " Recall signal traced. Results can be viewed on any communications console." : "" ]")
 
-/*
-/obj/docking_port/mobile/emergency/findTransitDock()
-	. = shuttle_master.getDock("emergency_transit")
-	if(.)	return .
-	return ..()
-*/
+/obj/docking_port/mobile/emergency/proc/is_hijacked()
+	for(var/mob/living/player in player_list)
+		if(player.mind)
+			if(player.stat != DEAD)
+				if(issilicon(player)) //Borgs are technically dead anyways
```

Comment:
Wait, does this mean that if the shuttle is piloted entirely by cyborgs, that the shuttle will head to the syndicate base :smirk: